specs/slack_spec.rb

```diff
@@ -0,0 +1,82 @@
+require_relative "test_helper"
+require "pry"
+
+# potential tests:
+# make sure main method makes a new slack instance?
+
+describe "Slack" do
+  describe "list users and channels" do
+    it "show a list of users" do
+      VCR.use_cassette("slack_query") do # <-- .yml filename
+        return_value = Slack.new
+        expect(return_value).must_be_kind_of Slack
+        expect(return_value.channels[0].slack_id).must_equal "CH41RMLP4"
+        expect(return_value.users[1].user_name).must_equal "cello.elle"
+      end
+    end
+  end
+
+  describe "select users and channels" do
+    it "select a user" do
+      VCR.use_cassette("slack_query") do
+        slack = Slack.new # <-- .yml filename
+        user_a = slack.select_user("UH53ZCBBR") # <-- this is "id" value for "kaseea"
+        user_b = slack.select_user("kaseea") # <-- this is "name" value for "kaseea"
+        # binding.pry
+        expect(user_a.user_id).must_equal "UH53ZCBBR"
+        expect(user_b.user_name).must_equal "kaseea"
+      end
+    end
+
+    it "select a channel" do
+      VCR.use_cassette("slack_query") do # <-- .yml filename
+        slack = Slack.new
+        channel_a = slack.select_channel("apiiiii")
+        channel_b = slack.select_channel("CH4BCTLHK") # <-- this is the "id" value for the general channel
+        expect(channel_a.name).must_equal "apiiiii"
+        expect(channel_b.name).must_equal "random"
+      end
+    end
+  end
+
+  describe "select users and channels" do
```

Comment:
This describe seems to include seeing details rather than select user and channels.

You should also test what happens when no user or channel is selected.  

You should test with both a user and channel.